archive.py

```diff
@@ -141,7 +146,8 @@ def write_stream_index(streams):
             sanitize_stream(s, streams[s]['id']),
             num_topics,
             '' if num_topics == 1 else 's'))
-    outfile.write('\n{% include ' + str(last_updated_file) + ' %}')
+    if jekyll_output:
+        outfile.write('\n{% include ' + str(last_updated_file) + ' %}')
```

Comment:
@Mark-Simulacrum what you want is for the `{%include ... %}` directive to be removed, right? Do you still want the `last_updated_file` info be in the output somehow?